examples/extension-api/hooks.py

```
@@ -24,7 +24,7 @@
 counter = 0
 
 
-def extension_hook(hook_name: str, context: dict) -> None:
+def extension_hook(hook_name: str, context: dict) -> Optional[dict]:
```

Pylint (3.3.7)

[refactor] Either all return statements in a function should return an expression, or none of them should.

(R1710)